README.md

```diff
@@ -6,9 +6,7 @@
 
 A great cloud needs great tools; we're excited to introduce *Azure CLI*, our next generation multi-platform command line experience for Azure.
 
-Take a test run now from Azure Cloud Shell!
-
-[![](https://shell.azure.com/images/launchcloudshell.png "Launch Azure Cloud Shell")](https://shell.azure.com)
+Take a test run now from [Azure Cloud Shell](https://portal.azure.com/#cloudshell/)!
```

Comment:
https://shell.azure.com now redirects to https://portal.azure.com/#cloudshell/

```
> curl --include https://shell.azure.com
HTTP/1.1 301 Moved Permanently
Content-Length: 160
Content-Type: text/html; charset=UTF-8
Location: https://portal.azure.com/#cloudshell/
Server: Microsoft-IIS/10.0
X-Powered-By: ASP.NET
Date: Fri, 12 Mar 2021 10:40:34 GMT

<head><title>Document Moved</title></head>
<body><h1>Object Moved</h1>This document may be found <a HREF="https://portal.azure.com/#cloudshell/">here</a></body>
```

So there is no need to use https://shell.azure.com.

The same applies to https://github.com/Azure/azure-powershell/pull/14531.

Reply:
The trailing slash (`/`) is against the [URL syntax](https://tools.ietf.org/html/rfc3986#section-3):

```
   The following are two example URIs and their component parts:

         foo://example.com:8042/over/there?name=ferret#nose
         \_/   \______________/\_________/ \_________/ \__/
          |           |            |            |        |
       scheme     authority       path        query   fragment
          |   _____________________|__
         / \ /                        \
         urn:example:animal:ferret:nose
``` 

So remove it.